### PR TITLE
Follow PyTorch Guide of updating Learning Rate

### PR DESCRIPTION
**Pull request by hyecheol123**
#1100 

I modified the code slightly, adding conditional statement to avoid calling decaying function on the first epoch

### CODE REVIEW

**Comment by junyanz** [comment]
Thanks for the PR. If this condition is added, it seems that the learning rate will not change during the first epoch of `contine_train`.  I am wondering if there is a workaround solution.  @hyecheol123  @taesungp

**Comment by hyecheol123** [comment]
Oh I did not tested for the case that start decreasing from the first epoch
(Maybe the case of n_epoch = 0 and n_epoch_decay = N?).

Is there any other case that I need to consider while dealing with learning rate decay?

**Comment by hyecheol123** [comment]
I did not considered continue training while writing that code and will take a look whether there is a good solution to handle it.

**Comment by hyecheol123** [comment]
I have one question regarding the starting epoch without continue_train option.

When continue_train does not specified, does it always starts train from epoch 1?
In this case, [discussion from the PyTorch forum](https://discuss.pytorch.org/t/loading-optimizer-dict-starts-training-from-initial-lr/36328) might be useful.

**Comment by junyanz** [comment]
The `epoch_count` is 1 by default. Without `continue_train`, it will start as 1.

**Comment by hyecheol123** [comment]
I will try to fix the code so that we can get proper learning rate decay for continued train.
